incubator/crate/templates/service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
```

Comment:
We've found it useful for performance reasons to split the CrateDB services into 2 components - one service (called crate-discovery) handles the discovery port `4300`, and one service to handle the rest (postgres, admin). This was because that the Postgres and Admin ports were fine to put behind a load balancer, but we saw performance issues when the cluster discovery ports were behind one. Just something to consider.

Reply:
Interesting insight. Before committing the split I'd like to test this out, but at the same time I'd like not to delay this PR any further. So I'm adding your suggestion as comment for now, since this could be adjusted later via a PR.